.circleci/config.yml

```diff
@@ -5,6 +5,6 @@ jobs:
       - image: integratedexperts/ci-builder
     steps:
       - checkout
-      - run: composer validate --strict || true
-      - run: .circleci/shellcheck.sh || true
-      - run: .circleci/phpcs.sh || true
+      - run: composer validate --strict --no-check-all
```

Comment:
added `--no-check-all` to not check version strictness of dependencies.

it is not clear why we have strict versions in `composer.json`, but this is a question for a separate issue.